benchmark/CMakeLists.txt

```diff
@@ -194,6 +194,10 @@ set(BENCH_DRIVER_LIBRARY_MODULES
 # Build Configuration
 #===-----------------------------------------------------------------------===#
 
+if (SWIFT_BENCHMARK_BUILT_STANDALONE)
+  set (SWIFT_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/..")
```

Comment:
Please remove the space between the `if`/`set` commands and the opening paren to conform to standard CMake styling.

Reply:
can we just get a cmake linter ; )

Reply:
There are such linters, I use one locally.

Reply:
That is not sufficient. We should install it on the bots, get the cmake code more standard to a good shape, and then make it an error to violate the linter.

Reply:
Here is my point... why should I even have to think about style in this way... It just wastes cycles in my brain. I would rather swift-ci just yell at me when I mess up.

Reply:
Adopting a CMake linter is something I hope we can do soon, but it will require **a lot** of cleanup and merge conflict resolution before we can start using it. We might also want to have linters run _before_ actually building for quick turnaround and less wasted engineering time.

Reply:
Actually, I've been meaning to do some PRs to cleanup the CMake. I have a little time now, no better time than the present.

Reply:
We can also make a special cmake linter job. If we are going to run this stuff early, we should do the same for the python lint.

Reply:
It'd be nice for the Python linter job to run not only the linter but also the Python unit-tests.